package.json

```diff
@@ -1561,25 +1554,24 @@
                     "type": "number",
                     "default": 60000,
                     "description": "%jupyter.configuration.jupyter.jupyterLaunchTimeout.description%",
-                    "scope": "resource"
+                    "scope": "application"
                 },
                 "jupyter.jupyterLaunchRetries": {
                     "type": "number",
                     "default": 3,
                     "description": "%jupyter.configuration.jupyter.jupyterLaunchRetries.description%",
-                    "scope": "resource"
+                    "scope": "application"
                 },
                 "jupyter.jupyterCommandLineArguments": {
                     "type": "array",
                     "default": [],
                     "description": "%jupyter.configuration.jupyter.jupyterCommandLineArguments.description%",
-                    "scope": "resource"
+                    "scope": "machine"
```

Comment:
This should not have been added to being with, this only applies in the case of non-zmq, which means this works on some machines and not others.
Also tomorrow if we ship zmq for all machines, then this property automatically becomes redunant and some feature that users might expect to work will not work 

Will file an issue to deprecate this